product_sale_price_security/security/product_price_security.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" ?>
+<odoo>
+    <record model="res.groups" id="group_product_sale_price">
+        <field name="name">display product prices</field>
+        <field name="category_id" ref="base.module_category_hidden" />
+        <field name="users" eval="[(4, ref('base.user_root'))]" />
+    </record>
```

Comment:
Can we make `sales_team.group_sale_salesman` inherit this group, to avoid confusion among the users (in that case, `sales_team` should be added as a dependency)?  The system admin would have to manually update the sales users and assign the new group to them, otherwise.

Reply:
I did not anticipate such a confusion from the viewpoint of the customer's requirements.
Thinking more generally, `sales_team` should be included the member of this group, I agree.
Thanks advice.